packages/react-vapor/package.json

```
@@ -43,6 +43,7 @@
         "diff2html": "2.5.0",
         "react-bootstrap": "0.32.4",
         "react-codemirror2": "5.1.0",
+        "react-color": "^2.17.3",
```

We use fixed dependencies version to avoid issue

```suggestion
        "react-color": "2.17.3",
```